lib/mapper.js

```diff
@@ -19,7 +19,7 @@ var time = require('unix-time');
 exports.track = function(msg, settings){
   var eventName = msg.event();
   var appEvents = settings.appEvents;
-  if (appEvents && appEvents[eventName]) {
+  if (appEvents && appEvents[eventName] && (appEvents[eventName]!=="null") ) {
```

Comment:
@f2prateek 
In UI: https://cloudup.com/c5cUAmbfZbl
In Code: http://screencast.com/t/rQ6v5lX7